File: app/src/main/java/com/wayfair/themoviedb/data/model/response/MultiSearch.java

package com.wayfair.themoviedb.data.model.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class MultiSearch {

    @SerializedName("page")
    @Expose
    public Integer page;
    @SerializedName("total_results")
    @Expose
    public Integer totalResults;
    @SerializedName("total_pages")
    @Expose
    public Integer totalPages;
    @SerializedName("results")
    @Expose
    public List<Result> results = null;

}

File: app/src/main/java/com/wayfair/themoviedb/data/model/response/MultiSearch.kt
package com.wayfair.themoviedb.data.model.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class MultiSearch {

    @SerializedName("page")
    @Expose
    var page: Int? = null
    @SerializedName("total_results")
    @Expose
    var totalResults: Int? = null
    @SerializedName("total_pages")
    @Expose
    var totalPages: Int? = null
    @SerializedName("results")
    @Expose
    var results: List<Result>? = null

}
